custom_components/bermuda/config_flow.py

```diff
@@ -104,14 +105,12 @@ def async_get_options_flow(config_entry):
     #     )
 
 
-class BermudaOptionsFlowHandler(config_entries.OptionsFlow):
+class BermudaOptionsFlowHandler(OptionsFlowWithConfigEntry):
     """Config flow options handler for bermuda."""
 
     def __init__(self, config_entry: ConfigEntry) -> None:
         """Initialize HACS options flow."""
-        super().__init__()
-        self.config_entry = config_entry
-        self.options = dict(config_entry.options)
+        super().__init__(config_entry)
```

Comment:
Looks like the super will save the config_entry and options into `self._config_entry` and `self._options`, so all our references will need to be updated to use the `_` prefix on those.

Reply:
![image](https://github.com/user-attachments/assets/48283c53-5b1a-4d7a-92b4-b9bba8b812f9)
They set .options and .config_entry as options

Reply:
SMH! I didn't see the property methods, my bad! 🤦🏼